packages/share_plus_web/README.md

```diff
@@ -0,0 +1,6 @@
+# Share Plus Web
+
+[![Flutter Community: sensors_plus_web](https://fluttercommunity.dev/_github/header/share_plus_web)](https://github.com/fluttercommunity/community)
+
+
+The Web implementation of [`share_plus`](https://pub.dev/packages/share_plus).
```

Comment:
We tried to make the README files consistent before the release. Could you use one of the other platform plugins' README as a template?

https://github.com/fluttercommunity/plus_plugins/blob/main/packages/share_plus_windows/README.md

Reply:
made it consistent with other packages readme